### PR TITLE
Cleanup for https://www.npmjs.com/package/devtools-launchpad

### PR DESCRIPTION
**Pull request by juliandescottes**
I might be the only one going to https://www.npmjs.com/package/devtools-launchpad but some of the links still point to debugger.html and the image is an issue.

### CODE REVIEW

File: packages/devtools-launchpad/package.json [hunk 1]
```diff
@@ -3,13 +3,13 @@
...
-    "url": "git://github.com/devtools-html/debugger.html.git",
+    "url": "git://github.com/devtools-html/devtools-core.git",
     "type": "git"
   },
   "bugs": {
-    "url": "https://github.com/devtools-html/debugger.html/issues"
+    "url": "https://github.com/devtools-html/devtools-core/issues"
   },
-  "homepage": "https://github.com/devtools-html/debugger.html#readme",
+  "homepage": "https://github.com/devtools-html/devtools-core#readme",
```
Comment (nchevobbe) [inline packages/devtools-launchpad/package.json]: We could put the devtools-launchpad one (https://github.com/devtools-html/devtools-core/tree/master/packages/devtools-launchpad) directly ?

File: packages/devtools-launchpad/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Launchpad
 
-<img width="1271" alt="debugger-screenshot" src="https://cloud.githubusercontent.com/assets/2134/19079518/bdb69580-8a08-11e6-909c-bc74e49bc395.png">
+![debugger-screenshot](https://cloud.githubusercontent.com/assets/2134/19079518/bdb69580-8a08-11e6-909c-bc74e49bc395.png)
```
Comment (nchevobbe) [inline packages/devtools-launchpad/README.md]: Maybe we could have a screenshot of the actual launchpad ?

Comment (juliandescottes) [inline packages/devtools-launchpad/README.md]: Yep sure. Let's use 

screen shot 2017-01-25 at 16 27 10-fullpage

File: packages/devtools-launchpad/package.json (hunk 1 above)
Comment (juliandescottes) [inline packages/devtools-launchpad/package.json]: Ah that was the intent, I keep mixing devtools-core and devtools-launchpad. thanks

**Comment by nchevobbe** [comment]
Now I guess you can `npm publish` :) ? I don't know what the process is, if there is some authentication or whatnot

**Comment by juliandescottes** [comment]
I don't have publish access for launchpad apparently :) that'll be for @jasonLaster

**Comment by jasonLaster** [comment]
Lets solve the publish rights question

**Comment by jasonLaster** [comment]
@nchevobbe what's your nmp account username?

**Comment by nchevobbe** [comment]
I don't have one 😂

Le 25 janv. 2017 5:33 PM, "Jason Laster" <notifications@github.com> a
écrit :

**Comment by nchevobbe** [comment]
okay @jasonLaster , here it is https://www.npmjs.com/~nchevobbe

**Pull request merged**